docs/gaia/gaia.rst

```diff
@@ -636,13 +639,13 @@ table named: user_<your_login_name>.'t'<job_id>::
   >>> from astroquery.gaia import Gaia
   >>> Gaia.login()
   >>> j1 = Gaia.launch_job_async("select top 10 * from gaiadr3.gaia_source")
-  >>> job = Gaia.upload_table_from_job(j1)
+  >>> Gaia.upload_table_from_job(job = j1)
   Created table 't1539932994481O' from job: '1539932994481O'.
 
 Now, you can query your table as follows (a full qualified table name must be provided,
 i.e.: *user_<your_login_name>.t<job_id>*)::
 
-  >>> full_qualified_table_name = 'user_<your_login_name>.t1539932994481O'
+  >>> full_qualified_table_name = 'user_<your_login_name>."t1710251325268O"'
```

Comment:
this looks weird, the `"`s are not in the table name in the narrative text in the line above

Reply:
This is required if the user wants to use a table name that contains capital letters. Otherwise, an error would be thrown. This is due to the TAP implementation at ESAC.

Reply:
My point is that the docs above says `user_<your_login_name>.t<job_id>`, without the quotation marks rather than `user_<your_login_name>."t<job_id>"`, so which one is it? If the latter, then the narrative above needs to be changed, too

Reply:
I see now what you mean. I've updated the text including the text user_<your_login_name>."t<job_id>" as well as a small explanation.